core/src/com/unciv/ui/cityscreen/ConstructionsTable.kt

```diff
@@ -300,7 +302,9 @@ class ConstructionsTable(val cityScreen: CityScreen) : Table(CameraStageBaseScre
                         selectedQueueEntry = -2
                         cityScreen.selectedConstruction = null
                     }
-                    if (!construction.shouldBeDisplayed(cityConstructions)) cityScreen.selectedConstruction = null
+                    if (!construction.shouldBeDisplayed(cityConstructions))
+                        cityScreen.selectedConstruction = null
+                    city.boughtPlaneThisRound = true
```

Comment:
It seems this flag will be set even if I buy not only the airplane - is it expected?

Reply:
oh good catch lol, dont know how I missed that one!